docs/_docs/history.md

```diff
@@ -1436,7 +1436,7 @@ note: This file is autogenerated. Edit /History.markdown instead.
 ### Site Enhancements
 {: #site-enhancements-v2-5-3}
 
-- Add `[@alfredxing](https://github.com/alfredxing)` to the `[@jekyll](https://github.com/jekyll)/core` team. :tada: ([#3218]({{ site.repository }}/issues/3218))
+- Add `@alfredxing` to the `@jekyll/core` team. :tada: ([#3218]({{ site.repository }}/issues/3218))
```

Comment:
Should probably remove the backticks around the name here, so it links properly.